bundles/org.openhab.binding.saicismart/src/main/resources/OH-INF/config/thing-config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:saicismart:vehicle">
+		<parameter name="vin" type="text" required="true">
+			<label>Vehicle Identification Number (VIN)</label>
+			<description>Unique VIN given by SAIC</description>
+		</parameter>
+		<parameter name="abrpUserToken" type="text" required="false">
+			<label>ABRP User Token</label>
+			<description>User Token for A Better Routeplanner.</description>
+		</parameter>
```

Comment:
Is it deliberate that the SAICiSMARTVehicleConfiguration.java has a timeout property , but it is not reflected here? So it is set, but not user changable? If you change it here, it should also be added to the readme.

Reply:
I will remove this as a configuration value and hard code the value in the code, as it is not for user configuration.